runtime/ops/encoding.rs

```diff
@@ -0,0 +1,129 @@
+//! Non-web API ops that deal with encoding and decoding of data.
+use deno_core::op;
+use deno_core::Extension;
+
+pub fn init() -> Extension {
+  Extension::builder()
+    .ops(vec![op_escape_html::decl()])
+    .build()
+}
+
+// TODO(@littledivy): Fast path for Latin1 strings
+#[op]
```

Comment:
should we also expose an override to allow passing in and returning a byte array? i would think there will be scenarios where you want to do escaping on byte arrays (e.g. resultsets coming from a database wire protocol) without converting to a JS string first?

Reply:
yup works totally, we could also make it Fast API'able (with a preallocated buffer & spill fallback)